_posts/2023-03-09-cockpit-287.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 287
+author: garrett
+date: '2023-03-09'
+tags: cockpit, podman
+slug: cockpit-287
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 287 and cockpit-podman 64:
+
+
+## Services: Pinned units need to be re-done
+
+The data storage for "pinned" systemd units has changed. After upgrading to this Cockpit version, you need to re-pin units.
+
+## Overview: Support for additional timeservers with chronyd
+
+Cockpit can now configure additional time servers when chronyd is used to synchronize the clock. Previously, this was only supported with systemd-timesyncd.
+
+![screenshot of support for additional timeservers with chronyd]({{ site.baseurl }}/images/287-overview-support-for-additional-timeservers-with-chronyd.png)
+
+
```

Comment:
We should remove the chronyd part. It was merged after the release. (And also needs a follow-up before next release: https://github.com/cockpit-project/cockpit/issues/18496)

```suggestion
```